core/src/main/scripts/importer/cbioportal_common.py

```diff
@@ -1085,13 +1085,16 @@ def get_db_cursor(portal_properties: PortalProperties):
 
     try:
         url_elements = dsnparse.parse(portal_properties.database_url)
+        # use SSL by default if "useSSL" is absent from the configuration file
+        useSSL = "useSSL" not in url_elements.query or url_elements.query["useSSL"] == "true"
         connection_kwargs = {
             "host": url_elements.host,
             "port": url_elements.port if url_elements.port is not None else 3306,
             "db": url_elements.paths[0],
             "user": portal_properties.database_user,
             "passwd": portal_properties.database_pw,
-            "ssl": "useSSL" not in url_elements.query or url_elements.query["useSSL"] == "true"
+            "ssl": useSSL,
```

Comment:
I think that we are changing the functionality somewhat with this refactoring. The original code was:

```
if portal_properties.database_use_ssl == 'true':
   connection_kwargs['ssl'] = {"ssl_mode": True}
   connection_kwargs['ssl_mode'] = 'REQUIRED'
```

Your code does not apply the same logic. Specifically, the value of field _ssl_ is not the same.